md/taxonomy.md

```diff
@@ -63,9 +66,6 @@
     * [6.x legacy forms](6-x-legacy-forms.md)
     * [Cache configuration and policy](cache-configuration-and-policy.md)
   * [Application descriptor](applicationCreation.md)
-  * [Profiles](_profiles.md)
```

Comment:
Should it be before 7.6 ?

Reply:
Profiles in the Studio is a 7.6 feature.
Before, it's only in the Portal.
I hope this answers the question.